doc/cascadia/JSONFragments.md

```diff
@@ -0,0 +1,123 @@
+
+- The namespace GUID: { 2BDE4A90 - D05F - 401C - 9492 - E40884EAD1D8 } 
+- The name of the profile to be updated 
+
+As a sanity check, a profile called 'Ubuntu' will get the generated GUID: { 2C4DE342 - 38B7 - 51CF - B940 - 2309A097F518 } 
+
+## Minimum requirements for settings added with fragments 
+
+There are some minimal restrictions on what can be added to user settings using JSON fragments:
+
+- For new profiles added via fragments, the new profile must, at a minimum, define a name for itself.
+- For new color schemes added via fragments, the new color scheme must define a name for itself, as well as define every color in the color table (i.e. the colors "black" through "brightYellow" in the example image above).
+
+## Where to place the JSON fragment files
+
+The location to place the JSON fragment files varies depending on the installation method of the application that wishes to place them.  
+
+### Microsoft Store applications 
+
+For applications installed through the Microsoft Store (or similar), the application must declare itself to be an app extension. More details on app extensions can be found in the [Microsoft Docs](https://docs.microsoft.com/windows/uwp/launch-resume/how-to-create-an-extension) and the necessary section is replicated here. The appxmanifest file of the package must include: 
+
+```xml
+<Package
+  ...
+  xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3"
+  IgnorableNamespaces="uap uap3 mp">
+  ...
+    <Applications>
+      <Application Id="App" ... >
+        ...
+        <Extensions>
+          ...
+          <uap3:Extension Category="windows.appExtension">
+            <uap3:AppExtension Name="com.microsoft.windows.terminal.settings"
+                               Id="<id>"
+                               PublicFolder="Public">
+            </uap3:AppExtension>
+          </uap3:Extension>
+        </Extensions>
+      </Application>
+    </Applications>
+    ...
+</Package>
+```
+
+Key things to note: 
+
+- The `"Name"` field must be `com.microsoft.windows.terminal.settings` for Windows Terminal to be able to detect the extension.
+- The `"Id"` field can be filled out as the developer wishes.
+- The `"PublicFolder"` field should have the name of the folder, relative to the package root, where the JSON files are stored (this folder is typically called "Public" but can be named something else if the developer wishes).
+- Inside the public folder, a subdirectory called "Fragments" should be created, and the JSON files should be stored in that subdirectory.
+
+### 'Traditionally' installed applications 
+
+For applications installed 'traditionally', there are 2 cases.
+
+The first is that the installation is for all the users on the system. In this case, the JSON files should be added to the folder:
+
+**C:\ProgramData\Microsoft\Windows Terminal\Fragments\\{app-name}**
```

Comment:
```suggestion
`C:\ProgramData\Microsoft\Windows Terminal\Fragments\{app-name}`
```

This might make more sense as a \<code> block? I can't put it in the "\`\`\`" code fences in a suggestion on github, but you get the idea